### PR TITLE
Created Split keyboard (markdown)

### PR DESCRIPTION
**Pull request by e3w2q**

### CODE REVIEW

File: Split-keyboard.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+kbd.init_pins(
+  [ 3, 4 ],      # row0, row1,... respectively
+  [ 29, 28, 27 ] # col0, col1,... respectively
+)
+kbd.add_layer :default  %i(
+  KC_1  KC_2  KC_3 
+  KC_4  KC_5  KC_6  
+)
+kbd.start!
+```
```
Comment (hasumikin) [inline Split-keyboard.md:41]: I might misunderstand your intention though, I don't think this works well.
The principle should be having the same keymap.rb on both sides.

**Review by hasumikin — COMMENTED** [review]
I also think Split-keyboard.md should contain a description about `Keyboad#set_anchor :right`.

File: Split-keyboard.md (hunk 1 above)
Comment (hasumikin) [reply Split-keyboard.md:41]: (like this https://github.com/picoruby/prk_crkbd/blob/main/keymap.rb )

Comment (e3w2q) [reply Split-keyboard.md:41]: Thank you!
Add a note to write kbd.set_anchor(:right) for USB connection on the right hand side.

**Pull request merged**